packages/web-frontend/src/App.js

```diff
@@ -48,6 +48,7 @@ class App extends Component {
 
     const { dispatch } = store;
     dispatch(fetchInitialData());
+    dispatch(findLoggedIn(true));
```

Comment:
We are using the 'shouldCloseDialog' key here... Is this fine or should we create a new, separate one? I think it is fine as is but wanted to flag it.

Reply:
As discussed, we should rename this variable to `loginType` and decouple login logic from UX logic. Each client of `findLoggedIn` can decide on how to manage the UX based on the `loginType`